read_store.py

```diff
@@ -1,38 +1,62 @@
 from app.src.stock.stocks_manager import *
 import json
+from datetime import *
+import threading
+from app.src.data_loader.trade_loader.websocket_utility import *
 #sorted_history = [(y.market_human_buy_power_ratio, x) for x, y in crawler.history.items()]
 #sorted_history.sort(reverse=True)
 
-START_BAZAR_HOUR = 9
-END_BAZAR_HOUR = 13
+START_BAZAR_HOUR = 0
```

Comment:
START_BAZAR_HOUR = 9
END_BAZAR_HOUR = 13.
They are fixed. just assign TESTING parameter in config, TRUE. The program will jump into the reading iteration.

Reply:
nice